lib/3scale_toolbox/commands/import_command/openapi/create_activedocs_step.rb

```diff
@@ -0,0 +1,52 @@
+module ThreeScaleToolbox
+  module Commands
+    module ImportCommand
+      module OpenAPI
+        class CreateActiveDocsStep
+          include Step
+
+          def call
+            active_doc = {
+              name: api_spec.title,
+              system_name: activedocs_system_name,
+              service_id: service.id,
+              body: resource.to_json,
+              description: api_spec.description
+            }
+
+            res = threescale_client.create_activedocs(active_doc)
+            # Make operation indempotent
+            unless res['errors'].nil?
```

Comment:
unless nil, so if errors? 

`if (errors = res['errors'])` and you can use just `errors` later in the branch.